### PR TITLE
Suggestions for Appendices chapter

### PR DESCRIPTION
**Pull request by gavinorland**

### CODE REVIEW

File: manuscript/documentation/01-readme.md [hunk 1]
```diff
@@ -26,3 +26,6 @@ TODO
 ## Conclusion
 
 TODO
+
+?> Maybe you could detail JSDocs,
+?> Maybe talk about Github.io and the ways these sites are repos.
```
Comment (sapegin) [inline manuscript/documentation/01-readme.md]: Do you mean GitHub pages? There should be a separate chapter about sites and everything.

File: manuscript/documentation/01-readme.md [hunk 2]
```diff
@@ -26,3 +26,6 @@ TODO
 ## Conclusion
 
 TODO
+
+?> Maybe you could detail JSDocs,
```
Comment (sapegin) [inline manuscript/documentation/01-readme.md]: Yup, but also typing and things like React Styleguidist. Not sure it should be here.

**Comment by sapegin** [comment]
Don’t forget to create a new branch for each pull request — this PR has changes from all previous PRs ;-)

File: manuscript/documentation/01-readme.md (hunk 1 above)
Comment (gavinorland) [reply manuscript/documentation/01-readme.md]: Yes, that's what I mean.

**Comment by gavinorland** [comment]
Right - not sure how that happened - I created a new branch for each set of commits then a PR for each branch.

**Comment by sapegin** [comment]
Looks like you were creating each new branch from previous branch, not from master.

**Pull request merged**